test/clipboard.test.ts

```diff
@@ -17,8 +18,9 @@ describe('Clipboard.', () => {
         }
     };
 
-    before(() => {
-        return conn().then((a: Fin) => fin = a);
+    before(async () => {
+        await cleanOpenRuntimes();
+        return fin = await conn();
```

Comment:
I'm going to approve the PR, but (being difficult) do we want to return an assignment?